seccomp.go

```diff
@@ -793,6 +793,30 @@ func (f *ScmpFilter) GetLogBit() (bool, error) {
 	return true, nil
 }
 
+// GetSsbBit returns the current state the SSB bit will be set to on the filter
+// being loaded, or an error if an issue was encountered retrieving the value.
+// The SSB bit tells the kernel that a seccomp user is not interested in enabling
+// Speculative Store Bypass mitigation.
+// The Log bit is only usable when libseccomp API level 4 or higher is
+// supported.
+func (f *ScmpFilter) GetSsbBit() (bool, error) {
```

Comment:
Can we just call this `GetSSB()`?  I dislike the forced lowercasing of acronyms in strict camel case.

Reply:
Changed